e2e/ctl_v3_lease_test.go

```diff
@@ -73,6 +75,33 @@ func leaseTestGrantLeasesList(cx ctlCtx) {
 	}
 }
 
+func leaseTestTimeToLiveExpired(cx ctlCtx) {
```

Comment:
Seems redundant with `authLeaseTestTimeToLiveExpired`. Could this be simplified by sharing code with `authLeaseTestTimeToLiveExpired`?